package.json

```diff
@@ -204,12 +204,12 @@
     "react-intl": "^2.4.0",
     "react-intl-po": "^2.1.1",
     "react-redux": "^4.4.5",
-    "react-redux-form": "^1.2.4",
     "react-router": "^3.0.4",
     "react-router-redux": "^4.0.6",
     "react-select": "^1.0.0-rc.5",
     "react-tooltip": "^3.3.0",
     "redux": "^3.6.0",
+    "redux-form": "^7.0.4",
```

Comment:
I'd like to get @go1dfish's approval on this redux-form pkg change and the new boilerplating you've used for the sign/verify forms.  If he likes it then we can open an issue to apply these same changes to every form.

Reply:
https://youtu.be/vYEXzx-TINc?t=10